src/main/java/edu/ucdavis/dss/ipa/entities/ExpenseItem.java

```diff
@@ -0,0 +1,148 @@
+    @Column(name = "Id", unique = true, nullable = false)
+    @JsonProperty
+    public long getId()
+    {
+        return this.id;
+    }
+
+    public void setId(long id)
+    {
+        this.id = id;
+    }
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "BudgetScenarioId", nullable = false)
+    @NotNull
+    @JsonIgnore
+    public BudgetScenario getBudgetScenario() {
+        return budgetScenario;
+    }
+
+    public void setBudgetScenario(BudgetScenario budgetScenario) {
+        this.budgetScenario = budgetScenario;
+    }
+
+    public BigDecimal getAmount() {
+        return amount;
+    }
+
+    public void setAmount(BigDecimal amount) {
+        this.amount = amount;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public String getTermCode() {
+        return termCode;
+    }
+
+    public void setTermCode(String termCode) {
+        this.termCode = termCode;
+    }
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "ExpenseItemTypeId", nullable = false)
+    @NotNull
+    @JsonIgnore
+    public ExpenseItemType getExpenseItemType() {
+        return expenseItemType;
+    }
+
+    public void setExpenseItemType(ExpenseItemType expenseItemType) {
+        this.expenseItemType = expenseItemType;
+    }
+
+    @JsonProperty("expenseItemTypeId")
+    @Transient
+    public long getExpenseItemTypeId() {
+        if(expenseItemType != null) {
+            return expenseItemType.getId();
+        } else {
+            return 0;
+        }
+    }
+
+    @JsonProperty("typeDescription")
+    @Transient
+    public String getExpenseItemTypeDescription() {
+        if(expenseItemType != null) {
+            return expenseItemType.getDescription();
+        } else {
+            return "";
+        }
+    }
+
+    @JsonProperty("termDescription")
+    @Transient
+    public String getTermDescription() {
+        if(termCode != null) {
+            String shortTermCode = termCode.substring(4,6);
+            switch (shortTermCode){
+                case "05":
+                    return "Summer Session 1";
+                case "06":
+                    return "Summer Special Session";
+                case "07":
+                    return "Summer Session 2";
+                case "08":
+                    return "Summer Quarter";
+                case "09":
+                    return "Fall Semester";
+                case "10":
+                    return "Fall Quarter";
+                case "01":
+                    return "Winter Quarter";
+                case "02":
+                    return "Spring Semester";
+                case "03":
+                    return "Spring Quarter";
+                default:
+                    return "";
+            }
+        } else {
+            return "";
+        }
+    }
+
+    @JsonProperty("budgetScenarioId")
```

Comment:
I don't know if this is any simpler or clearer, but it looks like you can annotate the BudgetScenario to serialize only the id

```java
@JsonProperty(value = "budgetScenarioId")
@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
@JsonIdentityReference(alwaysAsId = true)
```